src/db/schema.sql

```diff
@@ -0,0 +1,39 @@
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+
+CREATE TABLE IF NOT EXISTS public.teams (
+    team_name text PRIMARY KEY,
+    team_channel text NOT NULL,
```

Comment:
Change to include the category channel ID, voice channel ID and text channel ID as well as the team role ID